src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx

```diff
@@ -56,3 +101,21 @@ export const EmptyStateWithBoxAiDisabled: StoryObj<typeof MetadataSidebarRedesig
         },
     },
 };
+
+export const MetadataInstanceEditorWithDefinedTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+    render: () => {
+        const mockMetadataTemplate = {
+            ...mockCustomMetadataTemplate,
+            displayName: 'Template Name',
+            fields: mockTemplateFields,
+        };
+
+        return renderWithEditor(mockMetadataTemplate);
+    },
+};
+
+export const MetadataInstanceEditorWithCustomTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+    render: () => {
+        return renderWithEditor(mockCustomMetadataTemplate);
+    },
+};
```

Comment:
we should render the stories within the sidebar like how the other stories in this element are rendering